dev/wlp-jakartaee-transform/rules/jakarta-direct.properties

```diff
@@ -64,9 +64,9 @@ javax.servlet.jsp.jstl.sql.userName=jakarta.servlet.jsp.jstl.sql.userName
 javax.faces.validator.beanValidator.ValidatorFactory=jakarta.faces.validator.beanValidator.ValidatorFactory
 
 # Jakarta Server Pages TLD Locations
-/javax/servlet/jsp/resources/web-jsptaglibrary_1_1.dtd=/jakarta/servlet/jsp/resources/web-jsptaglibrary_1_1.dtd
-/javax/servlet/jsp/resources/web-jsptaglibrary_1_2.dtd=/jakarta/servlet/jsp/resources/web-jsptaglibrary_1_2.dtd
-/javax/servlet/jsp/resources/web-jsptaglibrary_2_0.xsd=/jakarta/servlet/jsp/resources/web-jsptaglibrary_2_0.xsd
+/javax/servlet/jsp/resources/web-jsptaglibrary_1_1.dtd=/jakarta/servlet/resources/web-jsptaglibrary_1_1.dtd
+/javax/servlet/jsp/resources/web-jsptaglibrary_1_2.dtd=/jakarta/servlet/resources/web-jsptaglibrary_1_2.dtd
+/javax/servlet/jsp/resources/web-jsptaglibrary_2_0.xsd=/jakarta/servlet/resources/web-jsptaglibrary_2_0.xsd
```

Comment:
Just to ensure I understand the pages resources now reside within the servlet bundle in the  `jakarta.servlet.resources` package?

Reply:
Correct